### ISSUE TITLE
"Does not work on Windows 7 due to SSL/TLS issues."

### ISSUE THREAD

**Issue by DumbJoe** [Jessecar96/SteamDesktopAuthenticator#968]
You're referring to the confirmations right which don't currently work for me, otherwise the steamguard codes work fine? Is there a way to fix the SSL/TLS issue? Or do you need a computer science degree to fix?

**Comment by rafaelrandom** [comment]
Download & Install .NET Framework 4.7.2 if you're using Windows 7.

**Comment by DumbJoe** [comment]
I have version 4.8 installed which is even newer than that, so technically that should cover what 4.7.2 requirements.....

**Comment by DumbJoe** [comment]
![image](https://github.com/Jessecar96/SteamDesktopAuthenticator/assets/36319788/42180f3b-19a1-43a2-8a19-b5dff3226598)

**Comment by DumbJoe** [comment]
> Well, if it doesn't even work with the .NET Framework installed, maybe it will work by upgrading to Windows 10 or 11.

That feels like side stepping onto a different lane when the original lane is blockaded.... Sure, it's a quick and easy solution (assuming the other lane(s) aren't blockaded) but doesn't quite solve *why* it is so in the first place.... Surely there's a way to destroy said blockade?

**Comment by rafaelrandom** [comment]
> > Well, if it doesn't even work with the .NET Framework installed, maybe it will work by upgrading to Windows 10 or 11.
> 
> That feels like side stepping onto a different lane when the original lane is blockaded.... Sure, it's a quick and easy solution (assuming the other lane(s) aren't blockaded) but doesn't quite solve *why* it is so in the first place.... Surely there's a way to destroy said blockade?

Probably not, what you can try to do is uninstall and reinstall the .NET Framework and if possible test on another pc to see if it works.

**Comment by Nooobish** [comment]
If you enable option "Periodically check for new confirmations and show a popup when they arrive" (File -> Settings) it will open small popup window in bottom right corner in which you will be able to accept trade offer.

**Comment by gortiik** [comment]
I just tested it and it works on Win7. Both market and trade confirmations.
Version 1.0.13

**Comment by clst** [comment]
True. The "periodically check" confirmation window still works :+1:

**Comment by Malanol** [comment]
> If you enable option "Periodically check for new confirmations and show a popup when they arrive" (File -> Settings) it will open small popup window in bottom right corner in which you will be able to accept trade offer.

Thanks a lot!
